src/vs/workbench/contrib/preferences/browser/settingsTree.ts

```diff
@@ -817,7 +822,7 @@ export abstract class AbstractSettingRenderer extends Disposable implements ITre
 			const disposables = new DisposableStore();
 			template.elementDisposables.add(disposables);
 			template.deprecationWarningElement.innerText = '';
-			template.deprecationWarningElement.appendChild(this.renderSettingMarkdown(element, element.setting.deprecationMessage!, template.elementDisposables));
+			template.deprecationWarningElement.appendChild(this.renderSettingMarkdown(element, element.setting.deprecationMessage!, disposables));
```

Comment:
I passed in `disposables` here to get it more consistent with the other call, which also passes in the newly-created `DisposableStore`.